File: loops/fizzbuzz.py
# Zoe Jimenez, FizzBuzz PYTHON
x = 1

while x <= 50:
    if x % 3 and x % 5:
        print("FizzBuzz")
    elif x % 5:
        print("Buzz")
    elif x % 3:
        print("Fizz")
    else:
        print(x)
    x+= 1

File: loops/fizzbuzz.c
// Zoe jimenez, FizzBuzz C
#include <stdio.h>




int main(void){


int x;

for(x=1;x<=50;x++){
     if(x % 5 == 0 && x % 3 == 0){
        printf("FizzBuzz\n");
    }else if(x % 5 == 0){
        printf("Buzz\n");
    }else if(x % 3 == 0){
        printf("Fizz\n");
    }else{
        printf("%d\n", x);
    }
}
}